src/pages/KubernetesError.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <div class="page-body">
+      <div class="error-header">
+        <img id="logo" src="../../resources/icons/logo-square-red@2x.png" />
+        <h2 data-test="k8s-error-header">
+          Kubernetes Error
+        </h2>
+      </div>
+      <div class="k8s-error">
+        <div class="error-part">
+          <h4>{{ titlePart }}</h4>
+          <pre white-space="pre-line">{{ mainMessage }}</pre>
```

Comment:
A long string in `mainMessage` can introduce some significant horizontal scrolling. I think it would look better and be easier to read if the text wrapped.

![image](https://user-images.githubusercontent.com/835961/150374454-84ce5bde-b025-4c78-a6cc-9184afd68384.png)

Reply:
I had anticipated that, but didn't implement it correctly. Fixed now.